prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.ts

```diff
@@ -185,6 +188,43 @@ export class PharmanetEnrolmentSummaryComponent extends BaseEnrolmentPage implem
 
     this.enrolment = this.enrolmentService.enrolment;
     this.isInitialEnrolment = this.enrolmentService.isInitialEnrolment;
+
+    this.careSettings.forEach(careSetting => {
```

Comment:
You could actually assign `this.careSettingConfigs` directly from the loop using `map`, and drop the `let`s and `break` and return the object literal directly from the switch cases since there are no defaults or shared value assignments.